src/app/notes/notes.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, OnInit } from '@angular/core';
+
+
+@Component({
+  selector: 'notes',
+  templateUrl: './notes.component.html',
+  styleUrls: ['./notes.component.css']
+})
+export class NotesComponent implements OnInit {
+
+	/*
+	* Data members for the notes component, title and content of the notes and patient ID
+	* for the corrosponding patient. 
+	*/
+
+	// public title: string;
+	public content: string;
+	public id: string;
+	
+
+  constructor() { 
+  	this.content=''; 
+  	this.id='';
+  }
+
+  public ngOnInit() {
+  }
+
+  // Loads a note from database based on patient ID
+	// for now loads in dummy text.
+	// TODO:Connect client side api
+	public loadNotes() {
+		let loadedNote: Object[] = this.getNotesQuery();
+
+		this.content = loadedNote['content'];
+		this.id = loadedNote['id'];
+	}
+
+	// Placeholder for CLIENT/API getNotes request
+	public getNotesQuery() {
+		let note: Object[] = [];
+		let tmpData: Object = JSON.parse(JSON.stringify(this.constructTestData()));
+
+		// Add validation check for JSON object before copying
+
+		note['content'] = tmpData['content'];
```

Comment:
Using the `['name']` syntax to bypass Typescript type checking is smelly. It would be better to create a type alias and assert that these objects are of that type, so you can use `note.content` instead.

Reply:
We will have a good approach to how to structure the notes data object tonight.

Reply:
I will look into it